src/components/Cart/Cart.js

```diff
@@ -11,6 +12,13 @@ import Content from '../Content';
 function Cart() {
   return (
     <Page title="Cart">
+      <InfoBanner
+        children={
```

Comment:
You don't have to pass children explicitly. It is everything between brackets by default. So it could be written:
**<InfoBanner>You have received <strong>free shipping</strong>!</InfoBanner>**